### PR TITLE
#3682 amend the presentation server readme for release 2.3

### PR DESCRIPTION
**Pull request by davidradl**

### CODE REVIEW

File: open-metadata-implementation/user-interfaces/presentation-server/README.md
Comment (grahamwallis) [inline open-metadata-implementation/user-interfaces/presentation-server/README.md:15]: I think this should say Integration View Services

File: open-metadata-implementation/user-interfaces/presentation-server/README.md
Comment (grahamwallis) [inline open-metadata-implementation/user-interfaces/presentation-server/README.md:18]: Should say Integration View Services

File: open-metadata-implementation/user-interfaces/presentation-server/README.md
Comment (grahamwallis) [inline open-metadata-implementation/user-interfaces/presentation-server/README.md:20]: allowing a user...

File: open-metadata-implementation/user-interfaces/presentation-server/README.md
Comment (grahamwallis) [inline open-metadata-implementation/user-interfaces/presentation-server/README.md:21]: allowing a user...

File: open-metadata-implementation/user-interfaces/presentation-server/README.md
Comment (grahamwallis) [inline open-metadata-implementation/user-interfaces/presentation-server/README.md:22]: allowing a user...

File: open-metadata-implementation/user-interfaces/presentation-server/README.md
Comment (grahamwallis) [inline open-metadata-implementation/user-interfaces/presentation-server/README.md:16]: I think this should say [Governance] Solution View Services

File: open-metadata-implementation/user-interfaces/presentation-server/README.md
Comment (grahamwallis) [inline open-metadata-implementation/user-interfaces/presentation-server/README.md:25]: [Governance] Solution View Services

File: open-metadata-implementation/user-interfaces/presentation-server/README.md
Comment (grahamwallis) [inline open-metadata-implementation/user-interfaces/presentation-server/README.md:32]: Should expand OMVS the first time - i.e. .... "Open Metadata View Services (OMVS) calls" ...

File: open-metadata-implementation/user-interfaces/presentation-server/README.md
Comment (grahamwallis) [inline open-metadata-implementation/user-interfaces/presentation-server/README.md:56]: We should also change "The each tenant's configuration.." to just "Each tenant's configuration.."

Comment (grahamwallis) [inline open-metadata-implementation/user-interfaces/presentation-server/README.md:56]: Need an apostrophe in "... that contains the tenant's serverName"

Also the environment variable for configuration presentation server is really called EGERIA_PRESENTATIONSERVER_SERVER_<tenant_name> 

It would be really nice to simplify the env variable name - it is kind of ugly.

Comment (grahamwallis) [inline open-metadata-implementation/user-interfaces/presentation-server/README.md:56]: Where we explain the content of the environment variable it would be nicer to describe the "remoteSevrerName" as the name of the View Server and to describe the "remoteURL" as the serverRootURL of the View Server.

Comment (grahamwallis) [inline open-metadata-implementation/user-interfaces/presentation-server/README.md:56]: Visual Studio Code should be uppercase C for Code

Also - I hate to mention it but do we need to acknowledge VSCode as a trademark???

File: open-metadata-implementation/user-interfaces/presentation-server/README.md
Comment (grahamwallis) [inline open-metadata-implementation/user-interfaces/presentation-server/README.md:58]: If someone wants to use a port other than 8091, where do they need to set it?

File: open-metadata-implementation/user-interfaces/presentation-server/README.md
Comment (grahamwallis) [inline open-metadata-implementation/user-interfaces/presentation-server/README.md:61]: I find that this doesn't work - I have to go to 'https://localhost:8091/aaa/login' rather than being able to stop at the tenant name.

**Review by grahamwallis — CHANGES_REQUESTED** [review]

**Review by planetf1 — APPROVED** [review]
I'm ok to merge this when ready -- I think it will be an area of ongoing improvement.
I'm reluctant to keep adding to 2.3 - we need to ship it, but if you have any updates tomorrow am let me know - if its just the .md file (low risk)

**Comment by planetf1** [comment]
I need to get an update into 2.3 near immediately. I need to get a hopefully final build and start the final tests & publishing

I propose to take this change (commits), and then make a few changes aligned with the comments above & create a 2.3 PR to be merged very soon. It may diverge slightly, as master could get more thorough updates.

**Comment by planetf1** [comment]
The readme (another PR) is only in master anyway.

**Review by planetf1 — COMMENTED** [review]
I've added my version of the updates - and text of my own in 6893c9cb67b5a5bc69c820b5665f6006834409bb

See https://github.com/odpi/egeria/pull/3733 for the 2.3 PR containing this.

I can push the commit to this master branch too if you like (I think it's a good idea?) then you could build any more on top - but I needed something quickly for 2.3

**Comment by planetf1** [comment]
I've added the updates I did for 2.3 branch

**Comment by planetf1** [comment]
Sorry! I've made a bad update which has confused the update, picking up 2.3 file changes.
My bad. I will fix it. do not merge.

**Comment by planetf1** [comment]
PR fixed

**Comment by planetf1** [comment]
suggest we merge in master - then iterate again through further improvements. at least it will get master up to the same docs as 2.3

**Pull request merged**
